### PR TITLE
Routing and single video page

### PR DESCRIPTION
**Pull request by gadgetfather**
- Added a single video page
- Added a basic routing
- Will add videos from the backend in the next PR
 ## Preview
image

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -7,6 +7,9 @@
   --category-pills-bg: rgb(242, 242, 242);
   --pills-border: rgb(217, 217, 217);
 }
+.link {
+  color: black;
```
Comment (b0llu) [inline src/App.css]: try to use variables

File: src/App.js [hunk 2]
```diff
@@ -1,14 +1,23 @@
...
       <MobileMenu />
+      <Routes>
+        <Route path="*" element={<h1>This under construction</h1>} />
+        <Route path="/" element={<HomePage />} />
+        <Route path="/watch-later" element={<h1>Watch later</h1>} />
+        <Route path="/liked-videos" element={<h1>Liked videos</h1>} />
+        <Route path="/history" element={<h1>History</h1>} />
+        <Route path="/playlist" element={<h1>Playlist</h1>} />
+        <Route path="/watch" element={<SingleVideoPage />} />
+      </Routes>
```
Comment (b0llu) [inline src/App.js:20]: Its good that you have a vision of your routes for next development

File: src/components/MobileMenu/MobileMenu.jsx [hunk 3]
```diff
@@ -1,20 +1,49 @@
...
+      <NavLink
+        className={({ isActive }) =>
+          isActive ? `link ${styles.active}` : "link"
+        }
+        to={"/liked-videos"}
+      >
+        <span title="Liked videos" className="material-icons">
+          thumb_up_off_alt
+        </span>
+      </NavLink>
```
Comment (b0llu) [inline src/components/MobileMenu/MobileMenu.jsx:46]: Great work on using NavLinks

**Review by b0llu — APPROVED** [review]
Great work on this PR. Suggested only one change everything else looks good to me.

File: src/components/VideoCard/VideoCard.jsx [hunk 4]
```diff
@@ -1,8 +1,13 @@
 import React from "react";
+import { useNavigate } from "react-router-dom";
 import * as styles from "./VideoCard.module.css";
 export function VideoCard() {
+  const navigate = useNavigate();
+  const handleVideo = () => {
```
Comment (AnkurChunekar) [inline src/components/VideoCard/VideoCard.jsx:6]: You can improve on namings here, instead of `handleVideo` you can change it to `navigateToWatch`

**Review by AnkurChunekar — APPROVED** [review]
Rest everything looks good. good job @gadgetfather

**Pull request merged**